frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -135,16 +145,19 @@ export class TransformTopologyData {
       deploymentConfig.kind = targetDeploymentsKind;
       const dcUID = _.get(deploymentConfig, 'metadata.uid');
 
-      const replicationController = this.getReplicationController(
+      const replicationControllers = this.getReplicationControllers(
         deploymentConfig,
         targetDeployment,
       );
-      const dcPods = this.getPods(replicationController);
+      const current = _.head(replicationControllers);
+      const previous = _.nth(replicationControllers, 1);
```

Comment:
I'm not certain this is right. If you have a failed rollout and then rollout again, `previous` here would be the failed rollout, not the last active replication controller.

It's possible we have this wrong in admin console, too.

Reply:
Thinking about this more, I believe it's an existing bug. If a rollout fails, we always show the latest revision, not the current, active RC (n-1). We'd want to show what's active here with some indication that the last rollout failed.

@christianvogt

Reply:
I think we should open a bug for this and address it in a follow on.

Reply:
@spadgett topology also needs to align with the overview codebase and we have a tech debt story to address that